source/includes/_appliance_repair_appliance_part_type_linker.md

```diff
@@ -10,6 +10,7 @@ __appliance_repair_appliance_part_type_linker_id__ <br><font color="DarkGray">_i
 __<a href="/#appliance-repair">appliance_repair_id</a>__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,foreign-key)</font> | 
 __<a href="/#old-appliance-part-type">old_appliance_part_type_id</a>__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,foreign-key)</font> | 
 __<a href="/#new-appliance-part-type">new_appliance_part_type_id</a>__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(foreign-key)</font> | 
+__repaired_parts_count__ <br><font color="DarkGray">_int_</font> <font color="Crimson"></font> | count of repaired parts
```

Comment:
Please change to this:
`__repaired_parts_count__  <br><font color="DarkGray">_int_</font> | Number of parts of this type replaced in this repair`